Assets/Scripts/Flying.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Flying : MonoBehaviour
+{
+    public Rigidbody2D rb;
+    float force = 10.0f;

```

Comment:
Should these variables be private or public?

```suggestion
    private float force = 10.0f;
    private bool hit = true;
```